### ISSUE TITLE
Warn users that files uploaded to Sia are not guaranteed to be retrievable yet

### ISSUE THREAD

**Issue by Fornax96** [NebulousLabs/Sia-UI#692]
A user on slack is claiming that he lost an archive of 10 years of photos because he didn't know that Sia is not stable yet. After he upgraded his client from 1.2.2 to 1.3.0 all his files became unavailable. This happened to me too when I upgraded to 1.3.0, I had to delete everything and start over because files that were uploaded before the upgrade weren't accessible anymore.

![image](https://user-images.githubusercontent.com/1247928/29162679-6ce22a10-7dba-11e7-960a-721064b3ba39.png)

I have no way to verify that this claim is true of course, but I myself have lost plenty of data through bugs in Sia. Of course I knew that I should keep local copies at all times and have not lost anything that is valuable to me.

I think this is a problem in Sia-UI. The UI gives an impression that it's meant as a replacement for traditional cloud services. As a temporary solution Sia-UI should warn users of possible instabilities in Sia and recommend keeping a local backup of files at all times. This can be done though the allowance dialog or the upload confirmation dialog.

**Comment by lukechampine** [comment]
Agreed, sadly. We do state this in a few places, but it's not enough, especially with an influx of new users. A big red warning in Sia-UI is probably sufficient.
